CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 Fix: Changing cygnus port does not work (#1698)
 [cygnus-commons][MongoBackend] Proper processing of mongo errors in create index operations (#1756)
+[cygnus-ngsi] Support for NGSIv2 notifications
```

Comment:
```suggestion
[cygnus-ngsi] Support for NGSIv2 notifications (#953)
```

Reply:
Fixed in [3e19cae](https://github.com/telefonicaid/fiware-cygnus/pull/1762/commits/3e19cae79a547c62d62434cdf9e841b40ede85b3)